config/HelixTaskWorkflowManagerConfig.properties

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (C) 20122 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+zk.layout.file.path=../config/zkLayout.json
+cluster.name=Ambry-corp
+# workflow.name=aggregationRecurrent
+# workflow.name=mysqlPartitionClassTriggerRecurrent
+workflow.name=mysqlTriggerRecurrent
+stats.reports.to.aggregate=PARTITION_CLASS_REPORT,ACCOUNT_REPORT
+delete.specified.workflow=false
+recurrent.interval.in.minutes=10800
```

Comment:
nit: add new line at the end of file.

Reply:
Also why are we adding linkedin specific config file in open source code?

Reply:
added newline but it doesn't show up, well actually it does. Without it , there'd be a red marker.

Reply:
nothing in this pr is linkedin specifc atm